Cargo.toml

```diff
@@ -8,5 +8,16 @@ description = "CCTalk protocol implementation for handling payment devices"
 edition = "2018"
 
 [dependencies]
-serial = "^0.4.0"
+serialport = "4"
 log = "0.4"
+
+[[example]]
+name = "coinacceptor"
+
+[[example]]
+name = "cctalk-host"
+
+[dev-dependencies]
+env_logger = "0.7"
+clap = "2.33.3"
+serialport = "4.0.1"
```

Comment:
why is this needed here? serialport is also in base dependencies

Reply:
Fixed